### ISSUE TITLE
Add New Member Projects

### ISSUE THREAD

**Issue by liam-middlebrook** [ComputerScienceHouse/CSHPublicSite#140]
The projects page is lacking of any exciting details about the cool projects that people work on here at CSH. This page should be overflowing with snippets about the various projects that members have worked on (wishfully it would be paginated from all the content that it has). 

What if we had members who felt comfortable list their major projects on here? I think that having a projects page the reflects on what house has actually been doing would be so much better to show to anyone who was interested in CSH (be they perspective students or potential donors).

**Comment by mbillow** [comment]
I have always been in favor of something if the like. My preference would be the page to highlight the best of the best and then there to be a large link to some type of CMS where people could actually write about their project.

**Comment by liam-middlebrook** [comment]
I personally would prefer small panels with the following
```
Name
2 liner description
GitHub link (this could have the full description)
```

I feel like that might be easier to get started with also (we don't have to setup CMS and we can keep it all static)

That being said either solution is better than what we currently have.

**Comment by sgreene570** [comment]
I agree with Liam on this one. Furthermore, I feel that some of the projects currently on the project page really don't belong there. "One Project Per Freshman" is not a project itself, its just a means of completing projects.  "Infosys" is hardly used and the feed hasn't been updated in a long time.  The "CSH Arcade" pictures are not accurate and do no explain what has been done to the machines we have at all.  I agree with Liam's idea of linking to external github links or other project documentation.  For what CSH actually accomplishes, the project page is very limited.

**Comment by rswiernik** [comment]
Fat fingered the close issue button. While I think having this issue open is valuable, I don't think further discussion without code changes is.

**Comment by brandonhudson** [comment]
Giving my thoughts on this now that we have a more developed PR plan - I think we should do a few things:

- Update the front page of the site
  - Include more recent project photos
  - Change 'More' to 'Read More'
- Add a projects tag/page to the new Jekeyll blog
- Update the projects page to include dynamic content
  - **Option 1:** Write a quick/dirty API to scrape the post markdown from the blog repo and include the first paragraph and a link to the page on the blog
  - **Option 2:** When the blog gets built, see if a new post was tagged as a project, and take a section of that content and put it somewhere else for easier reference. Then get the info from there.
    - JSON flat file
    - Database
   - Note: both of the requirements will need to standardize the way we do project posts to a certain extent (i.e. an image to use, a general/overview section for each one, etc.)

**Comment by stevenmirabito** [comment]
Ideally, I would like to see the blog and the public site incorporated into one Jekyll build that can then pull the most recent posts or whatever.

**Comment by devinmatte** [comment]
> Note: both of the options will need a somewhat standardized format for project posts (i.e. an image to use, a general/overview section for each one, etc.)

Luckily all blog posts will be prepended with the yaml information including an image. There's already logic in the jekyll blog for pulling the first paragraph of a post and using it as the preview text on the main landing page. This could honestly be very easily implemented

**Comment by devinmatte** [comment]
> Ideally, I would like to see the blog and the public site incorporated into one Jekyll build that can then pull the most recent posts or whatever.

Ideally, eventually, having the two be merged would be ideal. Jekyll has good support for templating as well as markdown to html conversion. A good major project would be converting the pub-site to Jekyll and integrating the blog with it more cohesively
